docs/deployment.md

```diff
@@ -1,5 +1,27 @@
+### Building for Deployment
+
+So, Xcode has some problems and will actually fail to compile the first time or two you try to build it (radars have been filed). To compensate for this, you need to run `fastlane build_for_deploy` until it works. 
```

Comment:
They're [working on it](https://twitter.com/jckarter/status/581507956769845250).